pyfar/classes/audio.py

```diff
@@ -578,7 +600,12 @@ def __init__(
         comment : str
             A comment related to `data`. The default is ``""``, which
             initializes an empty string.
-
```

Comment:
maybe we should also update the data docstring accordingly

Reply:
I added some documentation to the data docstring.